lib/assignment/range.js

```diff
@@ -14,6 +14,9 @@ function assignRange (topicPartition, groupMembers, callback) {
 
   const topicMemberMap = topicToMemberMap(groupMembers);
   for (var topic in topicMemberMap) {
+    if (!topicMemberMap.hasOwnProperty(topic)) {
```

Comment:
We can avoid this check if we update `topicMemberMap` instead and used `Object.create(null)` as the initiator to the `_.reduce`.